UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -122,6 +122,12 @@ public T GetSystem<T>()
 			return ActiveSystems.OfType<T>().FirstOrDefault();
 		}
 
+		public bool TryGetSystem<T>(out T system)
+		{
+			system = ActiveSystems.OfType<T>().FirstOrDefault();
+			return system != null;
```

Comment:
Codacy it's pretty sensible with this one

Reply:
"Use a comparison to 'default(T)' instead or add a constraint to 'T' so that it can't be a value type."